spec/lib/performance_platform/use_case/send_performance_report_spec.rb

```diff
@@ -153,7 +153,7 @@
               _timestamp: '2018-07-16T00:00:00+00:00',
               dataType: 'unique-users',
               period: 'month',
-              count: 12345
+              month_count: 12345
             }
```

Comment:
Do not worry, `day` variant is already tested in specs that didn't require a change.